src/exercises/exerciseTypes/audioExerciseOne/AudioExerciseOne.js

```diff
@@ -69,6 +72,9 @@ export default function AudioExerciseOne({
       bookmarksToStudy[0].id
     );
   }
+   if (!articleInfo) {
```

Comment:
Sorry, but there's something else that I don't understand.

`articleInfo` does not seem to be used anywhere in this exercise. 
Why are we loading it?

Reply:
Isn't it used for the loading animation at 75-77?

Reply:
sure; but why do you need the loading animation? 
you remove the loading animation, and the article info and nothing changes. or?

Reply:
Sure, I think we just added it because it was in the other exercises already, but I'll remove it